de_web_plugin.cpp

```diff
@@ -5274,8 +5275,9 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const deCONZ::
 
                 case IAS_ZONE_CLUSTER_ID:
                 {
-                    // Don't create ZHAalarm for this device using this cluster
-                    if (manufacturer.endsWith(QLatin1String("0yu2xgi")))
+                    // Don't create ZHAalarm for thoses device using this cluster
+                    if (manufacturer.endsWith(QLatin1String("0yu2xgi")) ||
+                        manufacturer.endsWith(QLatin1String("mdqxxnn")))
```

Comment:
Please add a comment so we can better identify the line later on.

`manufacturer.endsWith(QLatin1String("mdqxxnn"))) // TYZB01 light sensor`

Reply:
done